roles/openshift_node/templates/openshift.docker.node.service

```diff
@@ -34,6 +34,7 @@ ExecStart=/usr/bin/docker run --name {{ openshift.common.service_type }}-node \
   -v /lib/modules:/lib/modules -v /etc/origin/openvswitch:/etc/openvswitch \
   -v /etc/origin/sdn:/etc/openshift-sdn -v /var/lib/cni:/var/lib/cni \
   -v /etc/systemd/system:/host-etc/systemd/system -v /var/log:/var/log \
+  {% if openshift_use_nuage -%} $NUAGE_ADDTL_BIND_MOUNTS {% endif -%} \
```

Comment:
Add a default here so if the variable is not defined then it defaults to false.
`+  {% if openshift_use_nuage | default(false,true) -%} $NUAGE_ADDTL_BIND_MOUNTS {% endif -%} \`

Reply:
@sdodson Addressed the review comment. Thanks!